### PR TITLE
define more preciselly style

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: ytools/y2tool/rubocop_yast_style.yml [hunk 1]
```diff
@@ -39,3 +43,11 @@ Style/NegatedIf:
 # use "raise" instead of "fail"
 Style/SignalException:
   EnforcedStyle: only_raise
+
+# do not force %r
+Style/RegexpLiteral:
+  Enabled: false
+
+# Forbid and/or usage in conditionals
+Style/AndOr:
```
Comment (lslezak) [inline ytools/y2tool/rubocop_yast_style.yml]: Um, I'm just not sure about `and` and `or` keywords, I'd avoid them everywhere... It's too confusing IMHO.

I'd leave the RuboCop default (`always`), see https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L127

Comment (lslezak) [inline ytools/y2tool/rubocop_yast_style.yml]: The comment is wrong, it's the other way round (enabled in conditionals).

Comment (jreidinger) [inline ytools/y2tool/rubocop_yast_style.yml]: no, it should be forbidded there.
so you can do `a = b[:c] or raise "invalid data in #{b}"` but cannot do `if a and b`

Comment (lslezak) [inline ytools/y2tool/rubocop_yast_style.yml]: I'd leave the upstream default (banning them everywhere). IHMO it's confusing and used rarely.

**Comment by lslezak** [comment]
LGTM

**Pull request merged**